packages/frontend-2/lib/workspaces/graphql/mutations.ts

```diff
@@ -15,6 +15,21 @@ export const workspaceUpdateRoleMutation = graphql(`
   }
 `)
 
+export const workspaceUpdateSeatTypeMutation = graphql(`
```

Comment:
Minor naming convention:

`workspacesUpdateSeatTypeMutation` and `mutation WorkspacesUpdateSeatType`